cmd/finch/nerdctl.go

```diff
@@ -300,6 +330,15 @@ func handleEnvFile(fs afero.Fs, systemDeps NerdctlCommandSystemDeps, arg, arg2 s
 	return skip, envs, nil
 }
 
```

Comment:
Can you add some comment for the `getVolume` function. The intention of this functions returned values are not obvious. Adding a comment might help to understand the function easily. Also did you add any unit test for this function specifically?
What is the unit test coverage of the newly added code?

Reply:
The unit test coverage is almost 100%. 
I missed one line will add the test case for it.

```
nerdctlArgs = append(nerdctlArgs, arg)
```

For getVolume, we normally don't have separated unit tests for private function. It is part of public function and it is covered in the test of public function. In `make coverage`, it shows covered. But I will add comment.